tools/mti-utils/process_intensities.xml

```diff
@@ -0,0 +1,98 @@
+<tool id="cell_intensity_processing" name="Process single-cell intensities" version="0.0.1">
+    <description>Options to correct for expsure time, autofluorescence subtraction, or compute signal-to-background ratio.</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+        
+        ln -s '$quant_table' ./quant.csv &&
+
+        ln -s '$channel_csv' ./channels.csv &&
+
+        python '$script'
+
+    ]]></command>
+    <configfiles>
+        <configfile name = "script">
+import os
+import numpy as np
+import pandas as pd
+
+cwd = os.getcwd()
+quant = pd.read_csv(os.path.join(cwd, 'quant.csv'), index_col = 0)
+marker_df = pd.read_csv(os.path.join(cwd, 'channels.csv'))
+
+markers_to_normalize = marker_df['marker_name'].to_list()
+
+for marker in markers_to_normalize:
+
+#if $exp.correct_exposure:
+    exp_time = marker_df.loc[marker_df['marker_name'] == marker, '${exp.exp_col}'].values[0]
+    quant[marker] = quant[marker] / exp_time
+#end if
+
+#if $AF_method.select_method == 'subtract':
+    current_AF_channel = marker_df.loc[marker_df['marker_name'] == marker, '${AF_method.AF_col}'].values[0]
+    if current_AF_channel in markers_to_normalize:
+        quant[marker] = quant[marker] - quant[current_AF_channel]
+        quant[marker] = np.where(quant[marker] &lt; 0, 0, quant[marker])
+#elif $AF_method.select_method == 'SBR':
+    current_AF_channel = marker_df.loc[marker_df['marker_name'] == marker, '${AF_method.AF_col}'].values[0]
+    if current_AF_channel in markers_to_normalize:
+        quant[marker] = quant[marker] / quant[current_AF_channel]
+#end if
+
+quant.to_csv(os.path.join(cwd, 'processed_quant.csv'))
+        </configfile>
+    </configfiles>
+    <inputs>
+        <param name="quant_table" type="data" format="csv" label="Input quantification table (csv)" />
+        <param name="channel_csv" type="data" format="csv" label="Channel Metadata (csv)"/>
+        <conditional name="exp">
+            <param name="correct_exposure" type="boolean" checked="false" label="Correct for exposure time"/>
```

Comment:
conditionals should not be a boolean, but a select see here: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?highlight=version#booleans